Changelog main issue readme

RadValentin opened the pull request:
Cleaned up `README.md` and added new info according to issue #1

Commit:
add wireframe to readme #1

Commit:
rewrite Interface elements section to match current behavior #1

Commit:
finish readme changes closes #1

fix spelling errors, add list positioning wireframe, reference html
structure wiki page, add customization options section

RadValentin commented:
Please review the changes and let me know if anything else is needed
cc: @catalinmiron @skidding

ovidiuch commented:
> A widget that provides you with an easy way to send transparent status updates to your customers by tracking a branch on your GitHub repository. Closed issues show up as notifications inside the widget.

Not really content with this but we can change it at any time in the future. Try to express as much as possible with as few words as possible. E.g. "Use GitHub issues to communicate app updates directly to your customers."

> ![changelog wireframe](https://dl.dropboxusercontent.com/u/42934143/images/changelog.png)
- Link - This doesn't express anything. What does the label express? "New updates!" maybe. It's a button that _screams_
- Other - Why promote misc labels when we can enforce direct use-cases. I'd use "Feature" and "Bug", or even better "Fixed", "Improved", etc.
- Changelog (23) - changelog is a codename. I'm thinking "Updates ready (23)"

ovidiuch commented on README.md:
```diff
@@ -1,14 +1,18 @@
 **Behaviour**
-- when no unread notifications are present its styling should reflect this, possibly grayed out or with some default coloring.
-- when new notifications are received the icon styling changes to a different, brighter color. Possibly with a glow effect. In addition, a counter will show up displaying the number of new notifications.
+- when no unread notifications are present the icon won't be shown.
```
This sentence feels weird, shouldn't it be reversed? "The icon won't be shown until updates are ready."

PS. Try to start all sentences with a capital letter for consistency.

ovidiuch commented on README.md:
```diff
@@ -1,14 +1,18 @@
-- when no unread notifications are present its styling should reflect this, possibly grayed out or with some default coloring.
-- when new notifications are received the icon styling changes to a different, brighter color. Possibly with a glow effect. In addition, a counter will show up displaying the number of new notifications.
+- when no unread notifications are present the icon won't be shown.
+- when new notifications are received the icon becomes visible. In addition, a counter will show up displaying the number of new notifications.
```
If you nail the previous point right, you don't need to repeat yourself about the icon being visible only when update are ready. Instead, this point could only emphasize on one thing. "In addition to the visible icon, a small counter will also show up when updates arrive, displaying their count."

ovidiuch commented:
Finished review for now. Try to revise the doc with consideration to my comments and merge it. We'll have plenty of time to tweak it once we have a working prototype.

RadValentin committed:
update wireframe #1

Commit:
rewrite readme according to feedback #1

capitalize sentences, update widget description(shorter version),
rewrite notification icon behavior

RadValentin commented:
Updated the readme according to your feedback, let me know what you think.

cc: @skidding
